Chat/ClientApp/src/components/MainArea.tsx

```diff
@@ -3,10 +3,17 @@ import { Image } from '@fluentui/react';
 import defaultImg from '../assets/default.png';
 import { staticImageStyle, staticAreaStyle} from './styles/ChatScreen.styles';
 
-export default (props: any): JSX.Element => {
+export interface MainScreenProps {
+  updateContent(content: string): void;
+}
+
+export default (props: MainScreenProps): JSX.Element => {
   const imageProps = { src: defaultImg.toString() };
-  return (
-    <div className={staticAreaStyle}>
+  const { updateContent } = props;
```

Comment:
Not used?

Reply:
No not yet. I will have it in a part 2 PR but going to use it to change what `MainArea.tsx` displays I think. Or at least that is the plan I had. Open to thoughts